src/nvim/auevents.lua

```diff
@@ -161,6 +163,8 @@ return {
     LspTokenUpdate=true,
     RecordingEnter=true,
     RecordingLeave=true,
+    ReplayingEnter=true,
+    ReplayingLeave=true,
```

Comment:
```suggestion
    ReplayEnter=true,
    ReplayLeave=true,
```

Reply:
Sure, I'll change that. I've just went for the symmetry with `RecordingEnter` and `RecordingLeave`.